src/main/java/ladder/dto/Player.java

```diff
@@ -0,0 +1,82 @@
+package ladder.dto;
+
+import java.util.List;
+
+public class Player {
+    private String name;
+    private int location;
+
+    public Player(String name) {
+        validateName(name);
+        this.name = name;
+    }
+
+    public Player(String name, int location) {
+        validateName(name);
+        validateLocation(location);
+        this.name = name;
+        this.location = location;
+    }
+
+    public static void validateName(String name) {
+        if (name == null || name == "") {
```

Comment:
```suggestion
        if (name == null || name.equals("")) {
```

혹은 좀 더 간결하게, 아래와 같은 방식도 가능해요
```        
if ("".equals(name)) {
```

```